src/neorados/RADOS.cc

```diff
@@ -423,15 +432,15 @@ void Op::set_fadvise_nocache() {
     CEPH_OSD_OP_FLAG_FADVISE_NOCACHE);
 }
 
-void Op::cmpext(uint64_t off, bufferlist&& cmp_bl, std::size_t* s) {
+void Op::cmpext(uint64_t off, bufferlist cmp_bl, uint64_t* s) {
```

Comment:
Since this line is touched anyway, make it consistent with the header file:
```suggestion
void Op::cmpext(uint64_t off, bufferlist cmp_bl, uint64_t* unmatch) {
```